cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  const email = 'mlemeshev@i.ua';
+  const password = 'User1234';
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('input[type="email"]').type(email);
+    cy.get('input[type="password"]').type(password);
+    cy.get('.btn').click();
+    cy.get('#__next > nav > div > ul > li:nth-child(4) > a')
```

Comment:
It's better to avoid using complex selectors. They are difficult to read and maintain. If the structure of the page changes, the selector might break. Consider adding a data-test attribute or some other consistent identifier to the element you're trying to select.